node/src/types/json_compatibility/execution_result.rs

```diff
@@ -18,11 +19,37 @@ use casper_execution_engine::{
     },
     shared::{stored_value::StoredValue, transform::Transform as EngineTransform},
 };
-use casper_types::{CLValue, U128, U256, U512};
+use casper_types::{
+    bytesrepr::{self, FromBytes, ToBytes},
+    CLValue, U128, U256, U512,
+};
 
 #[cfg(test)]
 use crate::testing::TestRng;
 
+/// Constants to track operation serialization.
+pub const OP_READ_TAG: u8 = 1;
+pub const OP_WRITE_TAG: u8 = 2;
+pub const OP_ADD_TAG: u8 = 3;
+pub const OP_NOOP_TAG: u8 = 4;
+
+/// Constants to track Transform serialization.
+pub const IDENTITY: u8 = 0;
+pub const WRITE_CLVALUE: u8 = 1;
+pub const WRITE_ACCOUNT: u8 = 2;
+pub const WRITE_CONTRACT_WASM: u8 = 3;
+pub const WRITE_CONTRACT: u8 = 4;
+pub const WRITE_CONTRACT_PACKAGE: u8 = 5;
+pub const WRITE_DEPLOY_INFO: u8 = 6;
+pub const WRITE_TRANSFER: u8 = 7;
+pub const ADD_INT_I32: u8 = 8;
+pub const ADD_INT_U64: u8 = 9;
+pub const ADD_INT_U128: u8 = 10;
+pub const ADD_INT_U256: u8 = 11;
+pub const ADD_UINT_512: u8 = 12;
+pub const ADD_KEYS: u8 = 13;
+pub const FAILURE: u8 = 14;
```

Comment:
None of these should be `pub`.  Let's append `_TAG` to all of them as well.

Reply:
Have done so